README.md

```diff
@@ -1,44 +1,78 @@
-# What The Git
 
-## Introduction
+![What The Git Logo](public/logo.webp)
```

Comment:
Could you center the image? Also I'm not sure if adding the image in the repo is a good idea, ideally we should host the temporary logo somewhere.